sources/config_reader.c

```diff
@@ -327,9 +327,11 @@ od_config_reader_quantiles(od_config_reader_t *reader, char *value, double **qua
             return false;
         }
         *count += 1;
-        while (*c != ',') {
+        while (*c != 0 && *c != ',') {
```

Comment:
Hmm, it looks like you tried to check that "c" is nullptr but compare value "*c" instead of pointer "c".

Reply:
I've fixed that